db/migrate/20200709073219_add_permissions_to_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddPermissionsToTable < ActiveRecord::Migration[6.0]
+  def up
+    remove_column :provider_relationship_permissions, :make_decisions
+    remove_column :provider_relationship_permissions, :view_safeguarding_information
+    remove_column :provider_relationship_permissions, :type
+    add_column :provider_relationship_permissions, :training_provider_can_make_decisions, :boolean, null: false, default: false
+    add_column :provider_relationship_permissions, :training_provider_can_view_safeguarding_information, :boolean, null: false, default: false
+    add_column :provider_relationship_permissions, :ratifying_provider_can_make_decisions, :boolean, null: false, default: false
+    add_column :provider_relationship_permissions, :ratifying_provider_can_view_safeguarding_information, :boolean, null: false, default: false
+  end
+
+  def down
+    add_column :provider_relationship_permissions, :make_decisions, :boolean, null: false, default: false
+    add_column :provider_relationship_permissions, :view_safeguarding_information, :boolean, null: false, default: false
+    add_column :provider_relationship_permissions, :type, :string, null: false
+    remove_column :provider_relationship_permissions, :training_provider_can_make_decisions
+    remove_column :provider_relationship_permissions, :training_provider_can_view_safeguarding_information
+    remove_column :provider_relationship_permissions, :ratifying_provider_can_make_decisions
+    remove_column :provider_relationship_permissions, :ratifying_provider_can_view_safeguarding_information
+  end
+end
```

Comment:
I guess renaming columns is not an option because of STI (interpretation is dependent on `type`) and adding logic to preserve existing org relationships is not worth it, considering all existing relationships are test data. So we'll need to set up org relationships again in our dev environments, which should be fine but is hampered by the disappearance of `ProviderInterface::ProviderRelationshipPermissionsController`. Any chance we could bring it back?

Reply:
This should work, I think because it will effectively delete all the existing permissions but retain the training_provider/ratifying_provider relationships. If the sync happens to run during the migration it will fall over, but the subsequent sync will work.

@mnacos it will come back shortly — would be happy to discuss but my instinct is that the cost of refactoring that code is much greater than us having to type eg `ProviderRelationshipPermissions.find_by(training_provider: x, ratifying_provider: y).update(training_provider_can_manage_organisations: true)` for a few days

Reply:
Fair enough, we'll use the console for a few days.

I've verified this happens, existing relationships are maintained but permissions disappear (all org relationships fall back to View only).